### PR TITLE
MSR: slightly clean up diggernaut

### PR DESCRIPTION
**Pull request by Miepee**

### CODE REVIEW

File: randovania/games/samus_returns/logic_database/Area 6.txt [hunk 1]
```diff
@@ -714,8 +714,22 @@ Extra - asset_id: collision_camera_043
...
+
+> Before Diggernaut; Heals? False; Default Node
+  * Layers: default
+  * Extra - start_point_actor_name: ST_Checkpoint_009
+  > Door to Electric Escalade
+      After Area 6 - Diggernaut and Lay Power Bomb
+  > Event - Diggernaut
       All of the following:
-          # TODO: move this to event, also have a second look
+          # Damage Diggernauts Bomb Slots
```
Comment (dyceron) [inline randovania/games/samus_returns/logic_database/Area 6.txt:732]: Damage Diggernaut's Weak Points in the final phase

**Review by dyceron — CHANGES_REQUESTED** [review]
I wasn't sure if you planned on doing this in another PR, but if this is good for now, I'd rename the `Before DIggernaut` node to `Room Center` and then remove any connections that still go back to the right door.

File: randovania/games/samus_returns/logic_database/Area 6.txt (hunk 1 above)
Comment (Miepee) [reply randovania/games/samus_returns/logic_database/Area 6.txt:732]: its not just the final pahse tho? its also for the "spider up his legs and put a bomb into its slot"

Comment (dyceron) [reply randovania/games/samus_returns/logic_database/Area 6.txt:732]: Ah right. `Damage Diggernaut's Weak Points` is sufficient then

Comment (dyceron) [reply randovania/games/samus_returns/logic_database/Area 6.txt:732]: This wasn't addressed

Comment (Miepee) [reply randovania/games/samus_returns/logic_database/Area 6.txt:732]: whops didnt read good enough

**Review by dyceron — APPROVED** [review]

**Pull request merged**